spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/simple/SimpleProperties.java

```diff
@@ -30,14 +32,29 @@
 @ConfigurationProperties(prefix = "spring.metrics.export.simple")
 public class SimpleProperties {
 
+	/**
+	 * Enable publishing to the backend.
+	 */
 	private boolean enabled = true;
 
-	public boolean isEnabled() {
+	/**
+	 * The step size (reporting frequency) to use.
+	 */
+	private Duration step = Duration.ofSeconds(10);
+
+	public boolean getEnabled() {
```

Comment:
I think this should be`isEnabled()` as it was before

Reply:
In other classes, Phil called it getEnabled, so I was angling for consistency. I'll change them all to `is`

Reply:
I think the rule is `isEnabled()` for a `boolean` and `getEnabled()` for a `Boolean` but I might be wrong…